pkgs/development/python-modules/ibis-framework/default.nix

```diff
@@ -103,28 +113,14 @@ buildPythonPackage rec {
     rm setup.py
   '';
 
-  disabledTests = [
-    # These tests are broken upstream: https://github.com/ibis-project/ibis/issues/3291
-    "test_summary_numeric"
-    "test_summary_non_numeric"
-    "test_batting_most_hits"
-    "test_join_with_window_function"
-    "test_where_long"
-    "test_quantile_groupby"
-    "test_summary_numeric"
-    "test_summary_numeric_group_by"
-    "test_summary_non_numeric"
-    "test_searched_case_column"
-    "test_simple_case_column"
-    "test_summary_non_numeric_group_by"
-  ];
-
   pytestFlagsArray = [
-    "ibis/tests"
-    "ibis/backends/tests"
-    "ibis/backends/{${lib.concatStringsSep "," backends}}/tests"
+    "--dist=loadgroup"
+    "-m"
+    "'${lib.concatStringsSep " or " backends} or core'"
   ];
 
+  disabledTests = [ ];
+
```

Comment:
```suggestion
```

Reply:
for this to work, please change preCheck from `set -euo pipefail` to `set -eo pipefail`
the `disabledTests` fail, due to this bash setting `set -u` which will catch non referenced variables.